packages/clerk-js/src/ui/lazyModules/providers.tsx

```diff
@@ -80,7 +80,7 @@ type LazyModalRendererProps = React.PropsWithChildren<
     flowName?: FlowMetadata['flow'];
     startPath?: string;
     onClose?: ModalProps['handleClose'];
-    onExternalNavigate?: () => any;
+    onExternalNavigate: () => any;
```

Comment:
```suggestion
    onExternalNavigate: () => void;
```

to match the type below